tests/devops/Core/DevOps/StaticAnalyse/PHPStan/Rules/data/PropertyNativeTypeRule/PropertiesCorrectlyTyped.php

```diff
@@ -26,14 +19,12 @@ class PropertiesCorrectlyTyped
     public $callableProperty;
 
     /**
-     * @param string $promotedStringPropertyWithDeprecation
      * @param resource $promotedResourceProperty
      * @param callable $promotedCallableProperty
      */
     public function __construct(
         public string $promotedStringProperty,
-        /** @deprecated tag:v6.7.0 - Will be natively typed */
-        public $promotedStringPropertyWithDeprecation,
+        public string $promotedStringPropertyWithDeprecation,
```

Comment:
this property can be removed. as this is a test for the property type rule. but it is not a test case anymore now